package.json

```diff
@@ -45,7 +45,7 @@
     "zone.js": "^0.6.23"
   },
   "engines": {
-    "node": ">=0.10.0",
+    "node": ">=4.0.0",
```

Comment:
While you're at it, can you add ` <6.0` to indicate it won't run on node 6? (not sure what the actual syntax is, the `.0` might be unneeded)